recipe/meta.yaml

```
@@ -100,9 +102,14 @@ outputs:
       run:
         - isce3 ={{ version }}=*_cuda
     test:
-      imports:
-        - isce3
-        - isce3.cuda
+      {% set modules = [
+          "isce3",
+          "isce3.cuda",
+      ] %}
+      commands:
+        {% for m in modules %}
+        - python -c 'import pkgutil; import sys; sys.exit(pkgutil.find_loader("{module}") is None)'
```

Sorry had a small typo

```suggestion
        - python -c 'import pkgutil; import sys; sys.exit(pkgutil.find_loader("{{ module }}") is None)'
```